cmd/minikube/cmd/config/flags.go

```diff
@@ -25,3 +25,18 @@ import (
 func ClusterFlagValue() string {
 	return viper.GetString(config.ProfileName)
 }
+
+// ClusterKubernetesVersion returns the current Kubernetes version of the cluster
+func ClusterKubernetesVersion(clusterProfile string) string {
+	validProfiles, _, err := listProfiles()
+	if err != nil {
+		return ""
+	}
+
+	for _, profile := range validProfiles {
+		if profile.Name == clusterProfile {
+			return profile.Config.KubernetesConfig.KubernetesVersion
+		}
+	}
+	return ""
```

Comment:
```suggestion
	_, cc := mustload.Partial(clusterProfile)
	return cc.KubernetesConfig.KubernetesVersion
```
This should do the same thing in an easier way.